app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MessagesController < ApplicationController
+  def index
+    id = rand(1..Message.count)
+    @message = Message.find(id)
+    render json: @message
+  end
```

Comment:
- [ ] You also add the API file to make the messages_controller works on the react side. right now it render an errror

![Capture web_17-2-2022_13516_localhost](https://user-images.githubusercontent.com/61562626/154478735-665c9830-a1d3-4157-ae38-36723840a798.jpeg)

Reply:
Hi @dasileker, thanks for your suggestions! I think that is failing because you didn't load the db:seed and there are no messages available to show.

Reply:
However, I will add a response in case there are no messages so that error never appears.